contents/handbook/growth/sales/new-sales.md

```diff
@@ -0,0 +1,141 @@
+---
+title: New sales
+sidebar: Handbook
+showTitle: true
+---
+
+This is our playbook for managing new sales. These are customers who have contacted sales proactively - we don't do outbound sales. Some large customers have a very prescriptive internal process that you will need to follow, but for the vast majority of our customers you will be the one driving process (and even for customers with a defined process, don't just sit back and be a passenger!).
+
+## Sales process
+
+This is an overview for what you should actually be doing with a customer at each stage of the sales process. For details on how to log this in HubSpot specifically, visit the relevant [HubSpot flow](/handbook/growth/sales/crm#inbound-hands-on-pipeline). 
+
+**1. People contact us**
+
+Most people fill in the [contact sales](/contact-sales) form via the website. Some email sales@posthog.com or raise a Zendesk ticket.
+
+**2. We assign and qualify**
+
+Cameron reviews these and decides whether to route to [self-serve](/handbook/growth/sales/crm#inbound-self-serve-pipeline), schedule a demo, or pass to Simon (if they seem Very Large).  This handoff is currently a gut feel based on company size/revenue as Very Large customers may technically be considered low ICP per our [scoring](/handbook/growth/sales/icp). 
+
+**3. Initial demo**
+
+Some general principles:
+- Be genuinely helpful - our customers usually already have specific problems in mind, so help them solve those first rather 
+- Get to the point - our ICP is technical and particularly sensitive to sales BS
+- Don't [promise](/handbook/growth/sales/overview#enterprise-customers) things we can't/don't want to deliver
+- Get into the actual demo as quickly as possible - you should spend most of your time here, not presenting PostHog's vision
+
+We have a simple deck (seriously, it's like 5 slides) - ask someone on the Sales team for an invite to our Pitch account so you can create customized versions. 
+
+You generally want to make sure you cover:
+
+- Introduction, who is there, and what their roles are
+- Why they are here
+- Opportunity for them to ask up-front questions and get direct answers to them
+- Demo product features according to what they need
+- Consider complementary products, e.g. analytics + replay; analytics + warehouse; surveys + flags
+- More Q&A
+- Pricing - try and understand what their likely usage and spend is, as well as willingness to spend / commit to an annual plan
+- Next steps - this is really important, don't just end the demo with no clear follow up action
+
+At the end of the demo we might realise that they will be too small (<$20k) for us to go through our sales-led process, so you should route to self-serve. 
```

Comment:
do we want to keep this evaluation more open ended for now or would it help if we turn some of these into a template? e.g. on a scale of 1-3, customer interest level, potential for upsell, how well does current posthog features fit what they need

Reply:
I think templatifying this would be great! Let's leave it open ended for now (as we're currently doing) but then come back and update this once we have something in place? 

(I think this is good to have, but not massively high priority vs. other things)